Mathlib/Tactic/Module.lean

```diff
@@ -464,9 +464,9 @@ partial def parse (iM : Q(AddCommMonoid $M)) (x : Q($M)) :
     pure ⟨0, q(Nat), q(Nat.instSemiring), q(AddCommGroup.toNatModule), [], q(NF.zero_eq_eval $M)⟩
   /- anything else should be treated as an atom -/
   | _ =>
-    let k : ℕ ← AtomM.addAtom x
-    pure ⟨0, q(Nat), q(Nat.instSemiring), q(AddCommGroup.toNatModule), [((q(1), x), k)],
-      q(NF.atom_eq_eval $x)⟩
+    let (k, ⟨x', _⟩) ← AtomM.addAtomQ x
+    pure ⟨0, q(Nat), q(Nat.instSemiring), q(AddCommGroup.toNatModule), [((q(1), x'), k)],
+      (q(NF.atom_eq_eval $x'):)⟩
```

Comment:
The extra return value from `addAtomQ` means that the change on this line isn't needed any more

Reply:
Sorry, which line?

Reply:
Ah, I forget when you comment on a single line on github, it highlights the two lines above as well:
```suggestion
      q(NF.atom_eq_eval $x')⟩
```